docs/datagrid/style-by-css.md

```diff
@@ -28,9 +28,8 @@ The CSS is not included in the library by default, but you can import it like be
 @import url("https://code.highcharts.com/dashboards/css/datagrid.css");
 ```
 
-## Row/Cell height
-For performance reasons the height of every row, and hence cell, is static and long texts are truncated with an ellipsis.
-However, you can disable this by setting the [settings.rows.strictHeights](https://api.highcharts.com/dashboards/#interfaces/DataGrid_DataGridOptions.ColumnsSettings#strictHeights) option to `true`
+## Row height
+The height of each row is dynamic and based on the tallest cell. This allows for dynamic row height changes when adjusting column width. However, this can impact performance. You can disable this feature by setting the [settings.rows.strictHeights](https://api.highcharts.com/dashboards/#interfaces/DataGrid_DataGridOptions.ColumnsSettings#strictHeights) option to `true`.
```

Comment:
replace `tallest` with `highest`.

`It allows adjust row height when changing the column width`